recipes/reentry/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "reentry" %}
+{% set version = "1.0.2"%}
+{% set sha256 = "d981bf90c46a11819ff35bea227e4337019ae3b7719038b51e0e21180321c656"%}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  #fn: {{ name }}-{{ version }}.tar.gz
+  #url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  #sha256: {{ sha256 }}
+  git_url: https://github.com/DropD/reentry.git
+  git_rev: v1.0.2
+
+
+build:
+  number: 0
+  preserve_egg_dir: True
```

Comment:
Could you try removing the `preserve_egg_dir` and see if everything still works. See discussion here:
https://github.com/conda-forge/staged-recipes/issues/3056